common/src/main/java/com/microsoft/identity/common/AndroidPlatformComponents.java

```diff
@@ -242,6 +246,64 @@ public IMultiTypeNameValueStorage getFileStore(final @NonNull String storeName)
         return SharedPreferencesFileManager.getSharedPreferences(mContext, storeName, null);
     }
 
+    @Override
+    public INameValueStorage<String> getMultiProcessStringStore(final @NonNull String storeName) {
+        final SharedPreferences sharedPreferences = mContext.getSharedPreferences(storeName, Context.MODE_MULTI_PROCESS);
```

Comment:
We don't need the checking that's performed for some versions of Android configured by this flag.  We'll only be writing to this file from 1 process.  Rather than using this deprecated flag... let's just add a warning/comment to this code that writing to shared preferences from more than one process is not a good idea.  A problem exclusive to the broker for our work to date.

Adding link to docs: [Context.MODE_MULTI_PROCESS](https://developer.android.com/reference/android/content/Context#MODE_MULTI_PROCESS)

Reply:
If we don't use this.  I assume we can use the existing implementation.

Reply:
We cannot.

Reply:
" the file on disk will be checked for modification even if the shared preferences instance is already loaded in this process" - this is precisely what we will need.

Reply:
Have we tested to confirm that there is a problem here?  In other words... have we tested that process B is unable to read what process A has written?  I don't want to use a flag which is deprecated, because they discovered it doesn't work as intended...

Reply:
NOTE: Is the issue here about how to notify/update the file at runtime?

Reply:
I believe a cheap and dirty solution would be use an intent to launch an activity associated with the :auth process.  That activity doesn't have do have an UI per se....

Reply:
The issue is that the Authenticator process will write the file, and then it will be read in the auth process.  Authenticator will be updating on a daily cadence, so there will be writes at runtime.

Reply:
I've altered the test app to add a button that manipulates the flight manager in the android process instead of calling through the IPC api in order to see the results.  I put the change in the broker code review.  By altering what I was doing, I can establish that I do not read the writes of the other process unless I set the value to MODE_MULTI_PROCESS once I read.